generators/server/templates/Dockerfile.ejs

```diff
@@ -0,0 +1,18 @@
+FROM oracle/graalvm-ce:20.1.0-java11 as graalvm
+RUN gu install native-image
+
+COPY . /home/app/myapp
+WORKDIR /home/app/myapp
+
+<% if (buildTool === 'maven') { %>
+RUN native-image --no-server -cp target/*.jar
+<% } %>
+<% if (buildTool === 'gradle') { %>
+RUN native-image --no-server -cp build/libs/*-all.jar
```

Comment:
With GraalVM `20.2.0` you can remove `--no-server` because it is enabled by default.